src/aurora.ts

```diff
@@ -352,8 +357,11 @@ export class Aurora extends Construct {
       this.activityStreamArn = resource.getAttString('PhysicalResourceId');
     }
 
+    const passwordRotationIntervalInDays = props.passwordRotationIntervalInDays ?? 90;
```

Comment:
This doesn't match the documented default above. Also, we can remove it entirely if the default is `undefined`, which I think it should be.